tools/convert/converters/converter-csv.hpp

```diff
@@ -8,6 +8,7 @@
 #include <fstream>
 #include <map>
 #include <mutex>
+#include <array>
```

Comment:
needed?

Reply:
Used on line 53. Compiled before due to <array> being included unnecessarily in `rs_device.hpp`